feat(ui): settings screen revamp

HunnySajid opened the pull request:
Revamp Settings Screen with new theme


https://user-images.githubusercontent.com/19255438/209341100-c1882a33-0c58-46ef-ba2b-760fbc772a3f.mov

HunnySajid committed:
feat(ui): settings screen revamp

2byrds commented:
Good start @HunnySajid.  Could you make the username and avatar inputs look better for long urls, etc.  Also the save and toggle buttons don't look good.  I'm not sure how to improve them, or what to replace them with but could you take a shot at making them look more professional?
![image](https://user-images.githubusercontent.com/681493/209373175-49783340-c3a6-4f85-a422-cb7b09ca5269.png)

HunnySajid commented:
Hey @2byrds 
Instead of image url, I have added an image uploader component. Now user can upload or change image instead of pasting url. Also converted the toggle into buttons. 
What do you think about it?

https://user-images.githubusercontent.com/19255438/211221562-f11fea54-319f-4b44-9b13-e89f58c3d4ac.mov

2byrds commented:
Looks like a great start @HunnySajid . We'll need to add an issue for cropping images so they aren't stretched.
